pkgs/development/tools/ocaml/merlin/4.x.nix

```
@@ -95,6 +95,7 @@ buildDunePackage {
     description = "Editor-independent tool to ease the development of programs in OCaml";
     homepage = "https://github.com/ocaml/merlin";
     license = licenses.mit;
+    mainPackage = "ocamlmerlin";
```

```suggestion
    mainProgram = "ocamlmerlin";
```
Ideally the commit message would be equal to the new PR title, but I can fix that with a squash merge.

---

Whoops! Good catch